modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -0,0 +1,65 @@
+[id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
+= Configuration of JSON log data for default Elasticsearch
+
+When forwarding JSON logs to the default Elasticsearch instance, if the JSON log entries _have different formats_, you must create an index for each format.
+
+[IMPORTANT]
+====
+Forwarding different formats to the same index can cause type conflicts and cardinality problems.
+====
+
+To provide a different index for each format, you configure the `ClusterLogForwarder` custom resource. You use a structure type to construct the index name from a category, Kubernetes label, or input selector.
+
+.Structure types
+
+The `ClusterLogForwarder` provides the following structure types for use with the default Elasticsearch instance:
+
+* `structuredTypeName`: (string, optional) If `structuredTypeKey` is not set or its key is not present, use the value of `structuredTypeName` as the structured type.
```

Comment:
@alanconway Does this mean users can/should use `structuredTypeName` in combination with `structuredTypeKey` as a fallback if the structured log data doesn't contain the specified key?